openvdb/openvdb/points/PointSample.h

```diff
@@ -13,6 +13,7 @@
 #include <openvdb/util/NullInterrupter.h>
 #include <openvdb/thread/Threading.h>
 #include <openvdb/tools/Interpolation.h>
+#include <openvdb/util/Assert.h>
```

Comment:
Ehh I don't feel strongly either way - I've done the more "correct" thing here, so comment again if you want me to include it once and remove it elsewhere, otherwise I'll leave as is